README.md

```diff
@@ -7,9 +7,69 @@ See [Cap from the ORNL-CEES team](https://github.com/ORNL-CEES/Cap "Github - ORN
  +  graphical representation of sample output data generated from Dualfoil5.2
  +  tests to verify the functionality of the `battery` package
 
+## Installation
+ 1. Make sure you have downloaded the latest non-beta version of [docker](https://www.docker.com/products/overview "Get Docker")
```

Comment:
Don't even bother with the instruction on how to download/install of Docker. A working install Docker is a prerequisite.